calmdown.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+import json
+import os
+import re
+
+import discord
+from discord.ext import commands, tasks
+
+import utils
+from help.help import help
+
+"""
+    DISCORD_CALMDOWN_ROLE - Die Rollen-ID der "Stille Treppe"-Rolle.
+    DISCORD_CALMDOWN_FILE - Datendatei. Wenn diese noch nicht existiert wird sie angelegt.
+    DISCORD_DATE_TIME_FORMAT - Datumsformat für die interne Speicherung.
+"""
```

Comment:
Als Kommentar nicht unbedingt schlecht, am besten aber auch direkt in die .env.template Datei eintragen.